libs/model/src/policies/utils.ts

```diff
@@ -0,0 +1,189 @@
+import { Actor, logger } from '@hicommonwealth/core';
```

Comment:
just a naming question... are these policy utils or contest utils? ... let's think about how to name reusable command functions